msal/src/main/java/com/microsoft/identity/client/ISingleAccountPublicClientApplication.java

```diff
@@ -45,20 +47,78 @@
      * @param listener a callback to be invoked when the operation finishes.
```

Comment:
we should expose the isSharedDevice flag too.

Reply:
Done.